src/index.html

```diff
@@ -9,6 +9,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <div class="flex_container">
+        <a href="#home" class="logo_link">
+          <img
+            src="images/logo.png"
+            alt="logo"
+            class="logo flex_item"
+          >
+        </a>
+        <nav class="flex_item navigation">
+          <ul class="menu">
+            <li class="menu_item"><a href="*" class="nav_link is-active">Apple</a></li>
```

Comment:
Please format all code like this 
![image](https://user-images.githubusercontent.com/54936044/224473385-debd0bde-0717-486a-9b3e-1e36985f4df0.png)